ui/imports/shared/controls/WalletAccountListItem.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.15
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import AppLayouts.Wallet 1.0
+
+import utils 1.0
+
+StatusListItem {
+    id: root
+
+    property var modelData
```

Comment:
@Khushboo-dev-cpp make it `required`, cause setting `modelData` is obviously mandatory for this component, and you can get rid of checks like `!!modelData? modleData.some-prop : ""`

Reply:
Yeah, this doesn't solve the `null` problem

Reply:
Hey so the thing is, this component is used with a listView where it should and can never be null, but it is also used as a single component to represent the list item that was selected, in this case the object pointing to the modelData can turn to null due to a modelReset etc?

Reply:
nullish coalescing operator makes sense.

Reply:
If so, then you can simply expose name, emoji, color, other props you actually need instead `modelData`, that way this component will be fully reusable and doesn't rely on `modelData` and its props.

Reply:
ok but then these checks would have to happen outside, where we set this properties still then, right?

Reply:
yes but at another place it will be used as 

WalletAccountListItem {
    name: someProp && someProp.name ? someProp.name : ""
}

Reply:
That's ok, it depends on the place of usage.

`modelData` is used within delegate, in that case no need to check for `!!modelData`, just use `modelData.prop`
but if `prop` can be null, then just use `modelData.prop?? []` or whatever is needed.


> in this case the object pointing to the modelData can turn to null due to a modelReset etc?

Reading now this thread... are you sure about this?

Reply:
I think either null/undefined we start seeing errors for `cant ready property of undefined/null` in the logs